src/components/types.ts

```diff
@@ -0,0 +1,40 @@
+export interface Product {
+  id: string | number;
```

Comment:
Nechceme si to pri ukladani pohlidat a vzdy to ulozit jako string? 🙏

Reply:
abych byla zcela uprimna, vcera uz mi ruply nervy a nemela jsem energii zjistovat, ktere id je string a ktere number a kricelo to na me, takze rychly fix :D mam to v todo listu